prow/plugins/jira-linker/jira-linker.go

```diff
@@ -0,0 +1,106 @@
+package jira_linker
+
+import (
+	"regexp"
+	"strings"
+
+	"github.com/sirupsen/logrus"
+
+	"k8s.io/test-infra/prow/github"
+	"k8s.io/test-infra/prow/pluginhelp"
+	"k8s.io/test-infra/prow/plugins"
+)
+
+const (
+	pluginName  = "jira-linker"
+	jiraPrefix  = "jira/"
+	noJiraLabel = jiraPrefix + "no-ticket"
+)
+
+var (
+	jiraRegex = regexp.MustCompile("([A-Z]+)-\\d+")
```

Comment:
Could add a configurable list of project names to use inside a regex that does a `(FOO-\d+|BAR-\d+|...)`, and then fall back to this more general one?

Just a thought, I don't think I mind the fuzziness, and having a list means a thing to update in prow config periodically.

Reply:
I was discussing this with Sean just now, and figure it's a good way to avoid false-positives e.g. `ENG-593 Can't spell R-2D2 correctly` but I think this is something that can be added later if a need for it appears